playbooks/ceph.yml

```diff
@@ -158,6 +153,33 @@
       ansible.builtin.set_fact:
         ceph_hostname_var: 'ansible_hostname'
 
+    - name: Set IPv4 facts
+      when: ansible_all_ipv4_addresses | length > 0
+      ansible.builtin.set_fact:
+        ssh_network_range: 192.168.122.0/24
+        # storage_network_range: 172.18.0.0/24
+        storage_mgmt_network_range: 172.20.0.0/24
+        all_addresses: ansible_all_ipv4_addresses
+        ms_bind_ipv4: true
+        ms_bind_ipv6: false
+
+    - name: Set IPv6 facts
+      when: ansible_all_ipv4_addresses | length == 0
+      ansible.builtin.set_fact:
+        ssh_network_range: "2620:cf:cf:aaaa::/64"
+        # storage_network_range: "2620:cf:cf:cccc::/64"
+        storage_mgmt_network_range: "2620:cf:cf:dddd::/64"
+        all_addresses: ansible_all_ipv6_addresses
+        ms_bind_ipv4: false
+        ms_bind_ipv6: true
+
+    - name: Handle IPv6 job configuration
+      when:
+        - ansible_all_ipv4_addresses | length == 0
+        - not (cifmw_cephadm_ceph_spec_fqdn | default(false))
+      ansible.builtin.set_fact:
+        ceph_hostname_var: 'ansible_hostname'
```

Comment:
do we need this task for IPV6 explicitly? i see the task at L151 is doing the same.

Reply:
As per our slack discussion, this code had undergone enough testing,
and the confusion around cifmw_cephadm_ceph_spec_fqdn can be handled later on.